test/client_options_test.py

```diff
@@ -289,6 +297,32 @@ def test_config_no_section(config, capsys):
     assert repr("nosection") in captured.err
 
 
+def test_transfer_options(config):
+    parser = _options.Parser()
+    parser.add_sub_command("test", "help", lambda x: None)
+
+    args = parser.parse([
+        "test",
+        "-c", "all",
+        "--max-workers", "8",
+        "--buffer-size", "16m"
+    ])
+    assert args.max_workers == 8
+    assert args.buffer_size == 16 * MiB
+
+
+def test_transfer_options_disabled(config):
+    parser = _options.Parser()
+    parser.add_sub_command(
+        "test", "help", lambda x: None, transfer_options=False)
```

Comment:
Note for future change - we repeat this a lot in the tests, which is good since
this is important interface and we want to tests to be easy to read. But can we
change the defaults to it is easier to call this in the tests?

    parser.add_sub_command("test")

This will create a command with no help and function that does nothing.

Reply:
https://github.com/oVirt/ovirt-imageio/issues/120